File: source_code/CosmeticStoreAPI/CosmeticStoreAPI/Models/Product.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace CosmeticStoreAPI.Models
{
    public class Product
    {
        public string id { get; set; }
        public string name { get; set; }
        public string type { get; set; }
        public string producer { get; set; }
        public int quantity { get; set; }
        public int price { get; set; }
        public int description { get; set; }
        public bool status { get; set; }
    }
}

File: source_code/CosmeticFrontEnd/app/models/Product.js
var Product = function(
  id,
  name,
  type,
  producer,
  quantity,
  price,
  description
) {
    this.id = id;
    this.name = name;
    this.type = type;
    this.producer = producer;
    this.quantity = quantity;
    this.price = price;
    this.description = description;
};

var Product_ = function (id, name, type, price, img, quantity) {
  this.id = id;
  this.name = name;  
  this.type = type;
  this.price = price;
  this.img = img;  
  this.quantity = quantity;  
}
